ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -294,6 +294,39 @@ export default {
             cn: '死刑点名',
             ko: '탱버 대상자',
           },
+        };
+
+        const id = getHeadmarkerId(data, matches);
+
+        // Track tankbuster targets, regardless if this is on you or not.
+        // Use this to make more intelligent calls when the cast starts.
+        if (id === '00DA') {
+          data.formlessTargets = data.formlessTargets || [];
+          data.formlessTargets.push(matches.target);
+        }
+
+        // From here on out, any response is for the current player.
+        if (matches.target !== data.me)
+          return;
+
+        // Formless double tankbuster mechanic.
+        if (id === '00DA') {
+          if (data.role === 'tank')
+            return { alertText: output.formlessBusterAndSwap() };
+          // Not that you personally can do anything about it, but maybe this
+          // is your cue to yell on voice comms for cover.
+          return { alarmText: output.formlessBusterOnYOU() };
+        }
+      },
+    },
+    {
+      // Headmarkers are randomized, so use a generic headMarker regex with no criteria.
+      id: 'E12S Promise Headmarker - Junction Titan',
```

Comment:
```suggestion
      id: 'E12S Promise Junction Titan',
```

Reply:
Same conflict with this one and line 561:

```js
    {
      id: 'E12S Promise Junction Titan',
      netRegex: NetRegexes.tether({ id: titanTetherId, capture: false }),
```

Reply:
`E12S Promise Junction Titan Bombs` then?